lib/config/typescript.js

```diff
@@ -3,11 +3,17 @@ const merge = require('merge');
 module.exports = {
   extends: 'plugin:shopify/esnext',
 
-  plugins: ['typescript'],
+  plugins: [
+    '@typescript-eslint',
+    'eslint-comments',
```

Comment:
the `typescript` config extends from `esnext`, which in turn extends from `core` which adds the eslint-comments plugin (see https://github.com/Shopify/eslint-plugin-shopify/pull/233/files#diff-03d566093aa3fdf9491a9f677f0fa467R4) so I don't believe you need to specify this again here

Reply:
For some reason I did need to enable it again here :(